stripe-core/res/values/strings.xml

```diff
@@ -24,4 +24,12 @@
   <string name="stripe_address_label_state">State</string>
   <!-- Label for input requesting zip code, used for US based addreses -->
   <string name="stripe_address_label_zip_code">ZIP Code</string>
+  <!-- Label for a debit card, e.g. Visa Debit -->
+  <string name="stripe_link_card_type_debit">%s Debit</string>
+  <!-- Label for a credit card, e.g. Visa Credit -->
+  <string name="stripe_link_card_type_credit">%s Credit</string>
+  <!-- Label for a prepaid card, e.g. Visa Prepaid -->
+  <string name="stripe_link_card_type_prepaid">%s Prepaid</string>
+  <!-- Label for a card of unknown funding type, e.g. Visa Card -->
+  <string name="stripe_link_card_type_unknown">%s Card</string>
```

Comment:
Not sure how I feel about having these strings in `stripe-core`. I can move them elsewhere if anyone feels strongly about this, though that might mean moving the formatting logic into the UI layer.

Reply:
Moved them to `paymentsheet`.